src/components/Slider/BaseSlider/BaseSlider.scss

```
@@ -251,6 +251,10 @@ $rail-xl-size: 6px;
         }
     }
 
+    &_noMarks {
```

```suggestion
    &_no-marks {
```

Prefer not using camelCase in CSS